content/post/writing-a-book-remotely.md

```diff
@@ -0,0 +1,581 @@
+
+### Write Together, Not Alone
+
+Writing a book together has a lot of benefits. It motivates you to keep going when you would otherwise have quit when working on your own.
+
+We committed to delivering something every two weeks and had some high-level ideas regarding when the next release would be published. This goal-based work allowed us not to drift off and work towards the next milestone.
+
+Especially if it's your first book, writing it alone might feel like a big pile of work. Together, you get there step-by-step and can use the group's momentum.
+
+Furthermore, when co-authoring a book, you'll also improve your writing skills by getting constant feedback from other co-authors.
+
+We all had some background in writing technical content, which helped us a lot. Starting such a project with three people who had never written any technical blog before might be burdensome.
+
+### Use an Iterative Approach
+
+With an iterative approach, you don't have to lock yourself for a year in a wooden cabin in Norway and come back with a finished book.
+
+Start small. Deliver small increments, do baby steps, and get feedback early.
+
+We had a rough plan of what we wanted to cover but also adjusted a lot along the way.
+
+Leanpub is the perfect platform for this. Especially if you are a developer, you'll really enjoy writing your manuscript in Markdown. No annoying formatting issues inside Word when trying to center images or tables.
+
+Leanpub does the layout and file generation jobs for you. All you have to do is ship the manuscript.
+
+Pick a publisher that supports early access programs if you don't want to go towards the self-publishing route.
+
+You might be super convinced about your idea and think it's the most genius book ever. However, this doesn't necessarily reflect reality. Get some early feedback mechanisms in place that help you quantify the interest. A mailing list or a dedicated Twitter account can be such a tool.
+
+### Use Simple Tools
+
+Keep it simple. Use free plans and share the login with your co-authors if necessary.
+
+You can scale the tools and costs with your eBook's success.
+
+We purchased a Bootstrap-based HTML theme for our landing page for some bucks and adjusted it to our needs. During the project's inception phase, and while the eBook was not at 100%, we primarily used the landing page to attract potential readers. Our landing page had (and still has) a newsletter subscription form to keep potential readers around.
+
+We started with the free plan of ConvertKit and recently migrated to MailLite. The MailerLite pricing and plan structure was better for our needs.
+
+Some other tools/services that we used:
+- Grammarly for fundamental proofreading and typos
+- Writing the eBook in their editor of choice. I personally used IntelliJ IDEA most of the time.  I tweaked the editor settings to wrap the lines so that I can write more naturally without having to scroll endlessly horizontally or buy a curved monitor.
```

Comment:
```suggestion
- Writing the eBook in our editor of choice. I personally used IntelliJ IDEA most of the time. I tweaked the editor settings to wrap the lines so that I can write more naturally without having to scroll endlessly horizontally or buy a curved monitor.
```